Raaz/Core/Types/Tuple.hs

```diff
@@ -115,9 +115,9 @@ instance (V.Unbox a, EndianStore a, Dimension dim)
 
 
 -- | Construct a tuple by repeating a monadic action.
-repeatM :: (Functor m, Monad m, V.Unbox a, Dimension dim) => m a -> m (Tuple dim a)
```

Comment:
The FAM proposal was available starting with GHC-7.10. What was the corresponding version of package base.